content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -57,6 +69,41 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/master/con
 deployment.apps/my-nginx created
 ```
 
+
+### Group similar resources with PodList, ServiceList, and others
+
+Resources can be gathered into a list that is made up only of items of a particular type, then applied as needed. For example, you could create a list of pods in a `PodList`, services in a `ServiceList`, or secrets in a `SecretList`. The following is a list of supported list types:
+
+{{< table caption = "Available List types for grouping resources" >}}
+ | | | |
+|-------------------------------|--------------------------------|---------------------------|----------------|
+| APIServiceList | DaemonSetList | NamespaceList | ResourceQuotaList |
```

Comment:
I don't think you should list these out unless you're going to also tell users when they should use it. I don't think there's a great use case for these, it's very narrow. This documentation belongs with kubectl IMO, and if it goes in this document, I'd devote a footnote at most to it.

Reply:
E.g., you can just say something like "kubectl will also read some special List objects in addition to the more common document-separated YAML, click here for more detail" in a footnote.

Reply:
@lavalamp Thanks for the clarification. If the mentioned approach does work and it works only with `kubectl`, I'd suggest we put this to the kubectl section and add a pointer here.

Reply:
I removed my changes to this file.